pkg/controller/notification/notification_delivery_service_test.go

```diff
@@ -40,7 +41,11 @@ func (c *MockNotificationDeliveryServiceFactoryConfig) GetMailgunSenderEmail() s
 	return c.Mailgun.SenderEmail
 }
 
-func NewNotificationDeliveryServiceFactoryConfig(domain, apiKey, senderEmail, service string) NotificationDeliveryServiceFactoryConfig {
+func (c *MockNotificationDeliveryServiceFactoryConfig) GetMailgunReplyToEmail() string {
+	return c.Mailgun.ReplyToEmail
+}
+
+func NewNotificationDeliveryServiceFactoryConfig(domain, apiKey, senderEmail, replyToEmail, service string) NotificationDeliveryServiceFactoryConfig {
 	return &MockNotificationDeliveryServiceFactoryConfig{
 		Mailgun: MockMailgunConfiguration{Domain: domain, APIKey: apiKey, SenderEmail: senderEmail},
```

Comment:
the `replyToEmail` param is not used anywhere - it should be assigned to `MockMailgunConfiguration`:
```suggestion
func NewNotificationDeliveryServiceFactoryConfig(domain, apiKey, senderEmail, replyToEmail, service string) NotificationDeliveryServiceFactoryConfig {
	return &MockNotificationDeliveryServiceFactoryConfig{
		Mailgun: MockMailgunConfiguration{Domain: domain, APIKey: apiKey, SenderEmail: senderEmail, ReplyToEmail:replyToEmail},
```

Reply:
Fixed, good catch.